queue_job/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 
 {'name': 'Job Queue',
- 'version': '10.0.1.0.0',
+ 'version': '10.0.2.0.0',
```

Comment:
Well, it doesn't require a database migration, so perhaps `1.1.0` is sufficient.

I just wanted to call attention to the change of method into an attribute which could break code that depends on `queue_job`, but I can set it to `10.0.1.1.0` if you prefer.